src/index.html

```diff
@@ -207,18 +210,20 @@ <h3 class="filter__title">Price</h3>
           <h3 class="filter__title">Stock</h3>
           <div class="range">
             <div class="range__value">
-              <span>min</span>
-              <input type="number" class="input-min" value="1" max="65">
+              <label for="input-min">min</label>
+              <input type="number" id="input-min" class="input-min" value="1" max="65">

```

Comment:
можно атрибут `aria-label` добавить чтобы не добавлять видимые лейблы
```suggestion
              <input type="number" id="input-min" class="input-min" value="1" max="65" aria-label = 'Minimum'>

```